libs/datasets/sources/covid_county_data.py

```diff
@@ -72,14 +72,23 @@ def synthesize_test_metrics(cls, data: pd.DataFrame) -> Tuple[pd.DataFrame, pd.S
 
         # Make a copy to avoid modifying the argument when using mask with inplace=True.
         df = data.copy()
-        tests_and_cases = df.eval(
-            "negative_tests.isna() & positive_tests.isna() & total_tests.notna() & cases.notna()"
```

Comment:
negative_tests.isna() returns an `ndarray` boolean mask indicating which rows are missing negative_tests, but numexpr doesn't support `ndarray` and indeed the point of using eval() and numexpr is so it can execute the operation on a row-by-row basis and get better cache utilization, etc (_I think!_).  So to make this work with numexpr I had to rewrite it, which unfortunately ended up really ugly because numexpr doesn't have a `isnan()` function yet (coming in numexpr3 which has been stalled for over a year).  But if you want to see what that looked like, see here: https://github.com/covid-projections/covid-data-model/pull/775/commits/1bd07996fb3846a3d8498f5b29c11d7cd29a94f1#diff-bcae9bb794eb90c4637980f820d3df847a66c5392b15fdbd6577e8b278d916c6R75

But as I mentioned in the PR description that ended up being equivalent speed-wise (and this code isn't a bottleneck in the first place).

So I ended up dropping eval() entirely which was both faster and more readable.